CMakeLists.txt

```diff
@@ -94,9 +94,10 @@ endif()
 
 if (AWOOTER)
     include(FetchContent)
-    FetchContent_Declare(Corrosion GIT_REPOSITORY https://github.com/corrosion-rs/corrosion.git GIT_TAG v0.3.0)
+    FetchContent_Declare(Corrosion GIT_REPOSITORY https://github.com/corrosion-rs/corrosion.git GIT_TAG master)
```

Comment:
I think it'd be kind of inconvenient if some upstream change broke the build, which is why I specified a version rather than simply a branch.

Reply:
The Corrosion maintainer held back `corrosion_add_cxxbridge` from the v0.3.0 release for reasons I am not clear on (though given the number of issues that I have encountered and that have been opened against the repo, I can understand the reluctance). Supposedly it will land in a proper release by v0.4.0 which seems destined to arrive within a month or two.

In fixing the final cxx-build problems I may take a route that makes `corrosion_add_cxxbridge` temporarily irrelevant, which would be one way of getting rid of tagging master.